mobilecoind/clients/python/mob_client.py

```diff
@@ -234,6 +234,13 @@ def get_block_info(self, block):
         info = self.stub.GetBlockInfo(request)
         return info.key_image_count, info.txo_count
 
+    def get_block_details(self, block):
+        """ Returns detailed information for a ledger block.
+        """
+        request = api.GetBlockDetailsRequest(block=block)
+        details = self.stub.GetBlockDetails(request)
+        return details.hash, details.signature, details.txos, details.key_images 
```

Comment:
I think returning an unnamed tuple of four things is confusing, especially once we start including the block. I think it's better to just return the GRPC object since it already has everything in it and is easy to use.
This also makes it future-proof if we add more fields.